content/posts/December-19.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "December 19"
-date: 2019-12-06
+date: 2019-12-13
```

Comment:
Do we want to have post per `month` or per `week`?

Reply:
That's a good question.
On one hand, we want a new post each week.
On the other, with a post per week the [main blog page](https://packit.dev/posts) will soon be overcrowded and also the month names are better than week numbers to look through.

So that's why I use this hybrid.
Or now I have another idea: post per week and then later (let's say at the end of the month) squash them into months ? WDYT?

Reply:
- The current setup (post-per week grouped by month) is ok.
- But some archive looks good.
- Can we sort the posts on the main blog page in the reverse order?